dot/rpc/modules/author_test.go

```diff
@@ -323,7 +323,7 @@ func TestAuthorModule_PendingExtrinsics(t *testing.T) {
 
 func TestAuthorModule_InsertKey(t *testing.T) {
 	mockCoreAPI := &apimocks.CoreAPI{}
-	mockCoreAPI.On("InsertKey", mock.Anything).Return(nil)
+	mockCoreAPI.On("InsertKey", mock.Anything, mock.Anything).Return(nil)
```

Comment:
Can't you predict what should be received here? (same question for above `mock.AnythingOfType("string")`)